pom.xml

```diff
@@ -3,74 +3,46 @@
   xmlns="http://maven.apache.org/POM/4.0.0"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
                       http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <artifactId>docs.developer.ch.gov.uk</artifactId>
-
-  <build>
-    <plugins>
-
-      <plugin>
-        <artifactId>maven-surefire-plugin</artifactId>
-        <configuration>
-          <environmentVariables>
-            <CACHE_POOL_SIZE>10</CACHE_POOL_SIZE>
-            <CACHE_SERVER>localhost:1234</CACHE_SERVER>
-            <COOKIE_DOMAIN>.companieshouse.gov.uk</COOKIE_DOMAIN>
-            <COOKIE_NAME>abc</COOKIE_NAME>
-            <COOKIE_SECRET>iamasecret</COOKIE_SECRET>
-            <DEFAULT_SESSION_EXPIRATION>3600</DEFAULT_SESSION_EXPIRATION>
-            <HUMAN_LOG>1</HUMAN_LOG>
 
-            <cdn.url>//</cdn.url>
-            <chs.url>//</chs.url>
-
-            <account.url>//</account.url>
+  <modelVersion>4.0.0</modelVersion>
 
-            <createAccount.url>${home.url}/create-account</createAccount.url>
-            <developerGuidelines.url>${home.url}/developer-guidelines</developerGuidelines.url>
-            <gettingStarted.url>${home.url}/getting-started</gettingStarted.url>
-            <manageApplications.url>${home.url}/manage-applications</manageApplications.url>
-            <home.url>${home.url}</home.url>
-            <piwik.siteId>//</piwik.siteId>
-            <piwik.url>//</piwik.url>
+  <artifactId>docs.developer.ch.gov.uk</artifactId>
+  <version>unversioned</version>
+  <packaging>jar</packaging>
+  <name>docs.developer.ch.gov.uk</name>
+  <description>Static content for the developer hub,
+    such as "getting started" and "introduction" documentation
```

Comment:
Descrition can be changed to `Static general documentation content for...` deleting the `such as "getting started" and "introduction" documentation`.